### PR TITLE
update yarn.lock file after mochii upgrade; it was missing the caret

### PR DESCRIPTION
**Pull request by joshunger**
Associated Issue: #<issue number>

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* change 1
* change 2

### Test Plan

Tell us a little a bit about how you tested your patch.

Example test plan:

- [x] Command-P opens the panel
- [x] Clicking “+” opens the panel
- [x] Clicking a source navigates to the source
- [x] Clicking "x" closes the panel

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "flowtype.flow-for-vscode",
+    ]
+}
```
Comment (jasonLaster) [inline .vscode/extensions.json]: lets exclude these files

File: .gitignore [hunk 2]
```diff
@@ -5,7 +5,6 @@ configs/local.json
 npm-debug.log
 assets-error.log
 .idea/
-.vscode/
```
Comment (jasonLaster) [inline .gitignore]: I'm inclined to leave `.vscode` out. I'm not sure what other settings people use and it could conflict

Comment (joshunger) [reply .gitignore]: Whoops meant to do that in a separate checkin.  I'll remove and do another PR. Microsoft recommends checking it in see https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control.

**Comment by joshunger** [comment]
@jasonLaster fixed 👍

**Pull request merged**